src/pages/users/Checkout.js

```diff
@@ -0,0 +1,176 @@
+          <div className="form-row mt-3">
+            <div className="form-group col-md-3">
+              <label htmlFor="product">Product name</label>
+            </div>
+            <div className="form-group col-md-3">
+              <label htmlFor="quantity">Quantity</label>
+            </div>
+            <div className="form-group col-md-3">
+              <label htmlFor="unit-price">Unit price</label>
+            </div>
+            <div className="form-group col-md-3">
+              <label htmlFor="total-price">Total price</label>
+            </div>
+            {carts.map((item, index) => (
+              <>
+                <div className="form-group col-md-3" key={index}>
+                  <label htmlFor="product">{item.product_name}</label>
+                </div>
+                <div className="form-group col-md-3" key={index}>
+                  <label htmlFor="quantity">X{item.quantity}</label>
+                </div>
+                <div className="form-group col-md-3" key={index}>
+                  <label htmlFor="unit-price">X{item.unit_price}</label>
+                </div>
+                <div className="form-group col-md-3" key={index}>
+                  <label htmlFor="total-price">{item.total_price}</label>
+                </div>
+              </>
+            ))}
+          </div>
+        </div>
+        <div className="form-container">
+          <div className="form-group col-md-12 text-right">
+            <hr
+              style={{
+                borderWidth: "2px",
+                borderColor: "#C4C4C4",
+                marginTop: "10px"
+              }}
+            />
+            <div
+              className="form-inline"
+              style={{
+                display: "flex",
+                justifyContent: "space-between",
+                alignItems: "center"
+              }}
+            >
+              <h4 className="checkout-heading">Total of payment</h4>
+              <label htmlFor=""></label>
+              <input type="hidden" name="total_price" />
+            </div>
+          </div>
+          <div className="form-group col-md-12 text-right">
+            <div
+              className="form-inline"
+              style={{
+                display: "flex",
+                justifyContent: "space-between",
+                alignItems: "center"
+              }}
+            >
+              <h4 className="checkout-heading">Payment method</h4>
+              <select value={payment} onChange={handleSelect}
+                className="btn-sm form-select custom-select"
+                name="payment_method"
+              >
+               <option value="">--Choose an option--</option>
+                <option value="VNP">VNP</option>
+                <option value="COD">COD</option>
+              </select>
+            </div>
+            <hr
+              style={{
+                borderWidth: "2px",
+                borderColor: "#C4C4C4",
+                marginTop: "10px"
```

Comment:
Don't use style inline. Create css class for this one

Reply:
I have just resolved